apps/passport-client/src/zapp/ZappServer.ts

```diff
@@ -159,7 +159,8 @@ class ZupassPODRPC extends BaseZappServer implements ParcnetPODRPC {
   public async insert(collectionId: string, podData: PODData): Promise<void> {
     if (
       !this.getPermissions().INSERT_POD?.collections.includes(collectionId) ||
-      collectionId === "Devcon SEA"
+      collectionId === "Devcon SEA" ||
+      collectionId === "Email"
```

Comment:
nit -- maybe make this an array const? something like `VIRTUAL_COLLECTIONS` (or another name) ?